ddtrace/tracer/tracer.go

```diff
@@ -423,7 +426,9 @@ func (t *tracer) worker(tick <-chan time.Time) {
 			t.statsd.Incr("datadog.tracer.flush_triggered", []string{"reason:invoked"}, 1)
 			t.traceWriter.flush()
 			t.statsd.Flush()
-			t.stats.flushAndSend(time.Now(), withCurrentBucket)
+			if !t.config.tracingAsTransport {
+				t.stats.flushAndSend(time.Now(), withCurrentBucket)
```

Comment:
what happens with the data in `withCurrentBucket`?
should we mute the emitter side of things too (this place being the receiver)?

Reply:
the emitter side of things is muted via the change done to `canComputeStats()`

Reply:
nit for the next PR to come but I suggest then to update `t.stats` methods to noop them when this is disabled